src/assets/sw-utils.js

```diff
@@ -0,0 +1,946 @@
+var Mint = (function (exports) {
```

Comment:
Is defining `Mint` here is correct? Wouldn't that clash with the rest of the runtime?

Reply:
It won't because they are not loaded simultaneously, the `sw-utils.js` is loaded in the service worker while the `runtime.js` is loaded in the application itself.

Reply:
The _"standard runtime"_ is not included in the `service-worker` context, so it should not conflict with it.